.github/actions/security-scan/action.yaml

```diff
@@ -23,5 +17,4 @@ runs:
         severity: 'CRITICAL'
         exit-code: '1'
       env:
-        # TEMP FIX FOR: https://github.com/aquasecurity/trivy-action/issues/389
-        ACTIONS_RUNTIME_TOKEN: ${{ inputs.github-token }}
+        TRIVY_DB_REPOSITORY: ghcr.io/canonical/comsys-build-tools/trivy-db:2
```

Comment:
We should limit what repos have access to this DB to avoid abuse. I think we can do that after the package has been created.

Reply:
How do we do that?

Reply:
Ah sure